cli/azd/pkg/project/service_config.go

```diff
@@ -32,6 +32,8 @@ type ServiceConfig struct {
 	Infra provisioning.Options `yaml:"infra"`
 	// Hook configuration for service
 	Hooks map[string]*ext.HookConfig `yaml:"hooks,omitempty"`
+	// The optional dotnet project file if there're multiple project file in folder
+	DotnetProjectFile string `yaml:"dotnetProjectFile"`
```

Comment:
Since we decided to allow the user to put a filename in the project part of `azure.yaml`, let's remove this.